SECURITY.md

```diff
@@ -0,0 +1,17 @@
+# Security Policy
+
+## Supported Versions
+
+We always try to maintain the library secure and suggest our users to upgrade to the latest stable version. We realize that sometimes this is not possible.
+
+| Version | Supported          |
+| ------- | ------------------ |
+| 1.x     | :white_check_mark: |
+| < 1.0   | :x:                |
+
```

Comment:
The changes were synced to this commit https://github.com/graph-gophers/graphql-go/commit/64f8084368777e9e815b5191cef4e0c68fb07a12 committed on Jul 20 2022 and the latest stable version is [v1.4.0](https://github.com/graph-gophers/graphql-go/releases/tag/v1.4.0) which was released on April 11 2022 so this syncup with graphq-gophers/graphql-go is free of MaxDepth security vulnerability as this  vulnerability is detected in versions <v1.3.0

https://github.com/advisories/GHSA-mh3m-8c74-74xh